src/expertsystem/reaction/particle.py

```diff
@@ -243,60 +235,21 @@ def _repr_pretty_(self, p: PrettyPrinter, cycle: bool) -> None:
             p.text(")")
 
 
-class GellmannNishijima:
-    r"""Collection of conversion methods using Gell-Mann–Nishijima.
-
-    The methods in this class use the `Gell-Mann–Nishijima formula
-    <https://en.wikipedia.org/wiki/Gell-Mann%E2%80%93Nishijima_formula>`_:
-
-    .. math::
-        Q = I_3 + \frac{1}{2}(B+S+C+B'+T)
+ParticleWithSpin = Tuple[Particle, float]
 
-    where
-    :math:`Q` is charge (computed),
-    :math:`I_3` is `.Spin.projection` of `~.Particle.isospin`,
-    :math:`B` is `~.Particle.baryon_number`,
-    :math:`S` is `~.Particle.strangeness`,
-    :math:`C` is `~.Particle.charmness`,
-    :math:`B'` is `~.Particle.bottomness`, and
-    :math:`T` is `~.Particle.topness`.
-    """
-
-    @staticmethod
-    def compute_charge(state: Particle) -> Optional[float]:
-        """Compute charge using the Gell-Mann–Nishijima formula.
 
-        If isospin is not `None`, returns the value :math:`Q`: computed with
-        the `Gell-Mann–Nishijima formula <.GellmannNishijima>`.
-        """
-        if state.isospin is None:
-            return None
-        computed_charge = state.isospin.projection + 0.5 * (
-            state.baryon_number
-            + state.strangeness
-            + state.charmness
-            + state.bottomness
-            + state.topness
-        )
-        return computed_charge
-
-    @staticmethod
-    def compute_isospin_projection(  # pylint: disable=too-many-arguments
-        charge: float,
-        baryon_number: float,
-        strangeness: float,
-        charmness: float,
-        bottomness: float,
-        topness: float,
-    ) -> float:
-        """Compute isospin projection using the Gell-Mann–Nishijima formula.
-
-        See `~.GellmannNishijima.compute_charge`, but then computed for
-        :math:`I_3`.
-        """
-        return charge - 0.5 * (
-            baryon_number + strangeness + charmness + bottomness + topness
-        )
+def _compute_isospin_projection(  # pylint: disable=too-many-arguments
```

Comment:
I would just delete this, and directly add that line of code on the bottom

Reply:
Yes, it was just a remnant of the old code. Only reason to keep it was the unit test, but that test is not really useful.
--> 3d13ff8